handshake_ix.go

```diff
@@ -140,9 +140,6 @@ func ixHandshakeStage1(f *Interface, addr *udp.Addr, via *ViaSender, packet []by
 		},
 	}
 
-	hostinfo.Lock()
-	defer hostinfo.Unlock()
```

Comment:
The race detector might get upset about this, even if its okay in theory. I guess we can see what happens.

Reply:
It looks like its fine, at least running `make smoke-docker-race` shows no race detector complaints during the smoke test.